src/main/java/javaserver/Responders/FormResponder.java

```diff
@@ -0,0 +1,45 @@
+package javaserver.Responders;
+
+import javaserver.Form;
+import javaserver.HTTPStatusCode;
+import javaserver.Request;
+import javaserver.Response;
+
+public class FormResponder implements Responder {
+
+	private String[] supportedMethods;
+	private Form form;
+
+	public FormResponder(String[] supportedMethods, Form form) {
+		this.supportedMethods = supportedMethods;
+		this.form = form;
+	}
+
+	@Override
+	public Response getResponse(Request request) {
+		return new Response.ResponseBuilder(getStatusLine(request))
+      .body(getData(request))
```

Comment:
@catmclough Hmm.. might just be GitHub being weird. I think I had the same issue as well, and I wasn't sure why it was happening. It may be an issue with vim configurations. Are you using tabs or spaces as indentation?